hydra/_internal/utils.py

```diff
@@ -286,6 +286,23 @@ def _get_exec_command() -> str:
         return executable
 
 
+def _get_completion_help() -> str:
+    from hydra.core.plugins import Plugins
+    from hydra.plugins.completion_plugin import CompletionPlugin
+
+    completion_plugins = Plugins.instance().discover(CompletionPlugin)
+    completion_info: List[str] = []
+    for cls in completion_plugins:
+        plugin_cls = cast(Type[CompletionPlugin], cls)
```

Comment:
A perceptually cleaner way to do this is to use assert:
```suggestion
    for plugin_cls in completion_plugins:
        assert isinstance(plugin_cls, CompletionPlugin)
```